<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BooksManager</class>
 <widget class="QDialog" name="BooksManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>90</y>
     <width>501</width>
     <height>250</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTableWidget" name="alllisttableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>500</width>
       <height>210</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>书号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>作者</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>借出数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>在库数</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>图书库列表</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>在库信息</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="inkutableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>500</width>
       <height>210</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>书号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>作者</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>在库数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>可借阅</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>当前借出记录</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="outkutableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>500</width>
       <height>210</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>书号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>借阅人</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>借出时间</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>借出记录</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="recordtableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>500</width>
       <height>210</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>书号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>借阅人</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>借出时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>归还时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>归还状态</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>书库管理系统</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>200</width>
     <height>210</height>
    </rect>
   </property>
   <property name="title">
    <string>书库操作</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>28</y>
      <width>195</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>书号:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="bookidEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>书名:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="booknameEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>作者：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="bookauthorEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>总数：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="booktotalEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delButton">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>更新</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>200</width>
     <height>250</height>
    </rect>
   </property>
   <property name="title">
    <string>在库查询</string>
   </property>
   <widget class="QTextBrowser" name="infoBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>149</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="indexBox">
       <item>
        <property name="text">
         <string>书名</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>书号</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>作者</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>输入：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchEdit"/>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>infoBrowser</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>89</y>
     <width>61</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>功能</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>38</y>
      <width>61</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="alllistButton">
       <property name="text">
        <string>列表</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="inkuButton">
       <property name="text">
        <string>在库</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="outkuButton">
       <property name="text">
        <string>借出</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="recordButton">
       <property name="text">
        <string>记录</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="updatelistButton">
       <property name="text">
        <string>更新</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>系统时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="timelabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>60</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>系统时间</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="resulttableWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>501</width>
     <height>211</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>书号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>作者</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>在库数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>可借阅</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>21</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>查询结果</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
